### PR TITLE
reduce overhead

### PR DESCRIPTION
**Pull request by rettinghaus**
This PR speeds up the test suite by writing PNG files directly utilising the CairoSVG library.

### CODE REVIEW

**Review by lpugin — CHANGES_REQUESTED** [review]
This is good. However, we want to keep the SVG output. One idea with the diff process is also to compare the SVG trees at some point.

**Comment by rettinghaus** [comment]
Yeah, noticed this but ignored it. Now SVG _and_ PNG will be written in one go; still huge performance boost.

**Comment by lpugin** [comment]
Thanks. One more thing: does Cairo handle the VerovioText font properly?

**Comment by rettinghaus** [comment]
Seems like it does:
dir-004
lyric-005

**Comment by lpugin** [comment]
* I cannot see where VerovioText is used in this example, sorry. Please try with https://github.com/rism-ch/verovio.org/blob/gh-pages/_tests/lyric/lyric-007.mei and with https://github.com/rism-ch/verovio.org/blob/gh-pages/_tests/tempo/tempo-001.mei
* Make sure it builds

**Comment by rettinghaus** [comment]
CairoSVG output:
tempo-001
lyric-007

**Comment by lpugin** [comment]
Do you know why it fails ? https://github.com/rism-ch/verovio/pull/1924/checks?check_run_id=1650254844

**Comment by rettinghaus** [comment]
No, not really.

**Comment by rettinghaus** [comment]
Shouldn't we disable humdrum/pae/abc support for the test suite?

**Comment by lpugin** [comment]
> No, not really. Locally it works fine.

It does not for me
```
/Users/laurent/tmp/test-output-pr/accid/accid-001.png /Users/laurent/tmp/test-output-dev/accid/accid-001.png
Traceback (most recent call last):
  File "/Users/laurent/projects/verovio_develop/bindings/python/../../doc/test-suite-diff.py", line 114, in <module>
    diffValue = diffimg.diff(pngFile1, pngFile2, delete_diff_file=True)
  File "/usr/local/lib/python3.9/site-packages/diffimg/diff.py", line 28, in diff
    im2 = Image.open(im2_file)
  File "/usr/local/lib/python3.9/site-packages/PIL/Image.py", line 2891, in open
    fp = builtins.open(filename, "rb")
FileNotFoundError: [Errno 2] No such file or directory: '/Users/laurent/tmp/test-output-dev/accid/accid-001.png'
The file /Users/laurent/tmp/test-output-diff/index.html does not exist.
```

File: .github/workflows/tests_build.yml [hunk 1]
```diff
@@ -85,7 +84,6 @@ jobs:
           cd python
           make -j8
           python3 ../../doc/test-suite.py ${{ github.workspace }}/${{env.GH_PAGES_DIR}}/_tests ${{ github.workspace }}/${{ env.TEMP_DIR }}/${{ env.DEV_DIR }}/
-          ../../doc/test-suite-to-png.sh ${{ github.workspace }}/${{ env.TEMP_DIR }}/${{ env.DEV_DIR }}/
```
Comment (lpugin) [inline .github/workflows/tests_build.yml:88]: This is the problem. If you put it back it should work. It can then be removed in a subsequent PR.

Comment (rettinghaus) [reply .github/workflows/tests_build.yml]: Of course!

**Comment by rettinghaus** [comment]
@lpugin This is not my fault, is it? https://github.com/rism-ch/verovio/pull/1924/checks?check_run_id=1658874817

**Comment by lpugin** [comment]
> @lpugin This is not my fault, is it? https://github.com/rism-ch/verovio/pull/1924/checks?check_run_id=1658874817

No, this is probably a permission issue when adding the comment to the PR. I will look at it.

**Pull request merged**